docs/swagger/polaris-api.json

```diff
@@ -1417,6 +1417,26 @@
                     "duration": {
                         "type": "integer",
                         "example": 571
+                    },
```

Comment:
Thank you for not forgetting to update this!
I do have one minor request, could you keep the example values consistent with each other? The current ones are using a song from [this record](https://www.discogs.com/Stratovarius-Destiny/release/873481). Switching to a different song to have different lyricist vs composer is ok.